lib/rubocop/cop/commissioner.rb

```diff
@@ -24,30 +20,19 @@ def initialize(cops, forces = [], options = {})
         reset_errors
       end
 
-      # In the dynamically generated methods below, a call to `super` is used
+      # Create methods like :on_send, :on_super, etc. They will be called
```

Comment:
Does YARD have any annotations which allow us to generate documentation containing the methods created by manually listing them?

Reply:
Yes: http://www.rubydoc.info/gems/yard/file/docs/Tags.md#method. But I guess it'd be a bit unpleasant to see 95*N similar doc-macros lines in the file; and there is no way to use cycles in YARD, according to [this StackOverflow answer](https://stackoverflow.com/a/28415890/2610043).